pontoon/sync/vcs/models.py

```diff
@@ -37,6 +40,54 @@
 log = logging.getLogger(__name__)
 
 
+class DownloadTOMLParser(TOMLParser):
+    """
+    This wrapper is a workaround for the lack of the shared and persistent filesystem
+    on Heroku workers.
+    Related: https://bugzilla.mozilla.org/show_bug.cgi?id=1530988
+    """
+
+    def __init__(self, checkout_path, permalink_prefix):
+        self.checkout_path = checkout_path
+        self.permalink_prefix = permalink_prefix
+
+    def get_download_path(self, path):
+        """Return the directory in which the config file should be stored."""
+        return os.path.join(self.checkout_path, path)
+
+    def get_remote_path(self, path):
+        """Construct the link to the remote resource based on the local path."""
+        config_path = path.replace(self.checkout_path, "")
+
+        if "{locale_code}" in self.permalink_prefix:
+            return self.permalink_prefix.format(locale_code=config_path)
+        else:
+            return urljoin(self.permalink_prefix, config_path)
+
+    def get_project_config(self, path):
+        """Download the project config file and return its local path."""
+        download_path = self.get_download_path(path)
```

Comment:
Nit: I'd suggest we call it `local_path` since the comment calls it that way and download path could mean remote path as well.